pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -273,6 +273,9 @@ type PoolSpec struct {
 
 	// Parameters is a list of properties to enable on a given pool
 	Parameters map[string]string `json:"parameters,omitempty"`
+
+	// EnableRBDStats is used to enable gathering of statistics for all RBD images in the pool
+	EnableRBDStats bool `json:"enableRBDStats"`
```

Comment:
What is missing in the code generation? Something around this new field?

Reply:
Yes, I expected CRD to be updated with this new Spec. It didn't. Doing manually is error prone.

Reply:
The current code generation doesn't generate anything around these fields in this struct, it just references the type.